templates/zerver/help/search-for-messages.md

```diff
@@ -30,7 +30,7 @@ Here is the **full list of search operators**.
 * `is:private`: Search all your private messages.
 * `pm-with:ada@zulip.com`: Search 1-on-1 private messages between you and Ada.
 * `sender:ada@zulip.com`: Search messages sent by Ada.
-* `sender:me`: Search sent messages.
+* `sender:me`: [Search sent messages](/help/view-messages-sent-by-a-user#view-messages-sent-by-you).
```

Comment:
Let's make this "Search messages you've sent." for clarity.

Reply:
Sounds good!

<img width="323" alt="image" src="https://user-images.githubusercontent.com/2343554/197665219-3c3e44bb-38e0-4af1-ad39-a4737472efb3.png">